.github/workflows/cc_kbc.yml

```diff
@@ -47,6 +47,7 @@ jobs:
       - name: Build and install with cc_kbc feature
         run: |
           make KBC=cc_kbc && make install
+          make KBC=cc_kbc_tdx && make install
```

Comment:
The new `make` invocation doesn't fit the step's `name`.
Also below on the clippy step we don't use the `cc_kbc_tdx` feature.

Suggestsion: Consider adding a matrix dimension above (line 22):

```yaml
      matrix:
        rust:
          - stable
        kbc:
          - cc_kbc
          - cc_kbc_tdx
```

and then use `${{ matrix.kbc }}` in each of the steps to choose the current one.